front/reservationitem.form.php

```diff
@@ -90,16 +90,18 @@
     Html::back();
 } else if (isset($_POST["backToStock"])) {
```

Comment:
I think this entire case is a bad copy/paste from consumables/cartridge code. Reservations cannot be returned to stock.